src/Test.Utilities/VisualBasicSecurityCodeFixVerifier`2+Test.cs

```diff
@@ -17,6 +20,17 @@ public Test()
                 // These analyzers run on generated code by default.
                 TestBehaviors |= TestBehaviors.SkipGeneratedCodeCheck;
             }
+
+            protected override Project ApplyCompilationOptions(Project project)
+            {
+                var newProject = base.ApplyCompilationOptions(project);
+
+                var parseOptions = newProject.ParseOptions.WithFeatures(
+                    newProject.ParseOptions.Features.Concat(
```

Comment:
Same comment as C# case - this feature flag addition should move to the base type.

Reply:
Will do.